homeassistant/loader.py

```diff
@@ -900,6 +900,14 @@ def get_component(self) -> ComponentProtocol:
             )
             raise ImportError(f"Exception importing {self.pkg_path}") from err
 
+        if self.platform_exists("config"):
+            # Setting up a component always checks if the config
+            # platform exists. Since we may be running in the executor
+            # we will use this opportunity to cache the config platform
+            # as well.
+            with suppress(ImportError):
+                self.get_platform("config")
```

Comment:
We could also load all the integration platforms now to avoid blocking I/O later.. but I was really hoping to avoid teach the loader about which ones exist.  Something to think about for a future PR